README.md

```diff
@@ -10,32 +10,76 @@ MixedRealityToolkit-Unity uses code from the base [MixedRealityToolkit](https://
+| :-------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| To develop apps for mixed reality headsets, you need the Fall Creators Update.                                                                                  | The Unity engine is an easy way to get started building a mixed reality app.                                   | Visual Studio is used for code editing, deploying and building UWP app packages.                                           | Emulator allows you test your app without the device in a simulated environment.                                                                           |
 
 # Getting started with MRTK
-| ![Hololens Emulator(optional)](External/ReadMeImages/MRTK170802c_Short_22.png) Quick start | ![Hololens Emulator(optional)](External/ReadMeImages/MRTK170802c_Short_23.png) Contributing to this project |
-| :--------------------- | :----------------- |
+| ![Hololens Emulator(optional)](External/ReadMeImages/MRTK170802c_Short_22.png) Quick start                        | ![Hololens Emulator(optional)](External/ReadMeImages/MRTK170802c_Short_23.png) Contributing to this project |
+| :---------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------------- |
 | Please go over the [Getting started guide](GettingStarted.md) to learn more about getting off the ground quickly. | Please go over the [Contributing guidelines](CONTRIBUTING.md) to learn more about the process and thinking. |
 
+## New Features with Fall Creators Update!
+
+### Prerequisites:
+1. See [Development PC specs](https://developer.microsoft.com/en-us/windows/mixed-reality/install_the_tools#developing_for_immersive_headsets) for tips on developing for immersive headsets.
+2. [Holograms 100](https://developer.microsoft.com/en-us/windows/mixed-reality/holograms_100) has been updated with how to setup Windows Mixed Reality in your app.
+3. [How to navigate the Windows Mixed Reality home](https://developer.microsoft.com/en-us/windows/mixed-reality/navigating_the_windows_mixed_reality_home).
+4. Development PC needs to be on the Fall Creators Update (Build 1709)
+5. [Visual Studio 2017](https://www.visualstudio.com/downloads/).
+    1. Install the 10.0.16267.0 SDK via Visual Studio Installer.
```

Comment:
I believe the released Fall Creators Update SDK is 16299.

Reply:
Sorry I wrote the Version not the Build number.

Reply:
I mean line 48, where it says `10.0.16267.0`. That should be `10.0.16299.0`.

Reply:
Ahhh, let me double check what version is in VS Installer

Reply:
Fixed 😃